src/Sarif/Visitors/SarifCurrentToVersionOneVisitor.cs

```diff
@@ -50,7 +50,6 @@ internal AnnotatedCodeLocationVersionOne CreateAnnotatedCodeLocation(Location v2
                 {
                     Annotations = v2Location.Annotations?.Select(CreateAnnotation).ToList(),
                     FullyQualifiedLogicalName = v2Location.FullyQualifiedLogicalName,
-                    LogicalLocationKey = v2Location.FullyQualifiedLogicalName,
```

Comment:
>LogicalLocationKey [](start = 20, length = 18)

Why was this removed?

It's true that if you're going to keep it, you have to use `v2Location.FullyQualifiedLogicalName ?? v2Location.Name`, because `fullyQualifiedLogicalName` is optional if it's the same as `name`. But I'd've expected to see that fix, rather than to see the line removed.

Reply:
broken scenario. opened issue, marked 'urgent'

[https://github.com/Microsoft/sarif-sdk/issues/1180](https://github.com/Microsoft/sarif-sdk/issues/1180)

---
In reply to: [243867007](https://github.com/Microsoft/sarif-sdk/pull/1170#discussion_r243867007) [](ancestors = 243867007)